src/authService.js

```diff
@@ -0,0 +1,38 @@
+// authService.js
+import axios from 'axios';
+
+export const isAuthenticated = async () => {
+  try {
+    const token = localStorage.getItem('token');
+    const formData = new FormData();
+    formData.append('token', token);
+
+    const formDataObject = {};
+    formData.forEach((value, key) => {
```

Comment:
I dont understand this code, why use formData if in the end you just need to convert it to object and pass to api ?